doc/policies/authentication.rst

```diff
@@ -155,11 +155,15 @@ type: string
 This is the text that is sent via SMS to the user trying to
 authenticate with an SMS token.
 You can use the tags *<otp>* and *<serial>*.
+Longer texts must be enclosed in single quotes.
```

Comment:
I think *longer* is not quite correct. You must include it in quotes, if the text contains spaces, right?
A text like

    Hello_my_dear_your_OTP_value_is:<otp>

would still work. (SPACE usually splits multiple values in a policy action value)

Reply:
Done